qdrant-landing/content/articles/qdrant-1.3.x.md

```diff
@@ -0,0 +1,204 @@
+
+#### Set the oversampling factor via query:
+
+Here is how you can configure the oversampling factor - define how many extra vectors should be pre-selected using the quantized index, and then re-scored using original vectors.  
+
+```http
+POST /collections/{collection_name}/points/search
+
+{
+  "params": {
+    "quantization": {
+      "ignore": false,
+      "rescore": true,
+      "oversampling": 2.4
+    }
+  },
+  "vector": [0.2, 0.1, 0.9, 0.7],
+  "limit": 100
+}
+```
+
+```python
+from qdrant_client import QdrantClient
+from qdrant_client.http import models
+
+client = QdrantClient("localhost", port=6333)
+
+client.search(
+    collection_name="{collection_name}",
+    query_vector=[0.2, 0.1, 0.9, 0.7],
+    search_params=models.SearchParams(
+        quantization=models.QuantizationSearchParams(
+            ignore=False,
+            rescore=True,
+            oversampling=2.4
+        )
+    )
+)
+```
+
+In this case, if `oversampling` is 2.4 and `limit` is 100, then 240 vectors will be pre-selected using quantized index, and then the top 100 vectors will be returned after re-scoring.
+
+As you can see from the example above, this parameter is set during the query. This is a flexible method that will let you tune query accuracy. While the index is not changed, you can decide how many points you want to retrieve using quantized vectors.
+
+### Asychronous I/O interface
+
+Going forward, we will support the `io_uring` asychnronous interface for storage devices on Linux-based systems. Since its introduction, `io_uring` has been proven to speed up slow-disk deployments wherever the OS syscall overhead gets too high and the software becomes IO bound.
+
+<aside role="status">This experimental feature works on Linux kernels > 5.4 </aside>
+
+`io_uring` uses a ring buffer data structure to queue and manage I/O operations asynchronously, minimizing the need for context switches and reducing overhead. It supports features like submission and completion queues, event-driven I/O, and batch processing, allowing for efficient handling of I/O events and reducing the CPU overhead associated with traditional I/O.
+
+#### Enable async storage interface from the storage configuration file:
+
+```python
+storage:
+	# enable the async scorer which uses io_uring
+	async_scorer: true
+```
+You can return to the mmap based backend by either deleting the `async_scorer` entry or setting the value to `false`.
+
+This operation generates a lot of disk IO, so it is a prime candidate for possible improvements.
+Please keep in mind that this feature is experimental and that the interface may change in further versions.
```

Comment:
To my mind, that section lacks a sentence about possible improvements. How much faster may it be? Did the dev team make any benchmarks?

It would be also great to mention which AWS network disks may benefit from it, as many people were using the wrong ones.

Reply:
@llogiq I think we should add a small table of benchmark results and some perfomance comments....wdyt?